### ISSUE TITLE
Generator not working with Node 13.2.0 and npm 6.12.1

### ISSUE THREAD

**Issue by berquist** [EricCrosson/generator-emacs-package#19]
Using Node 13.2.0 and npm 6.12.1, this happens just after entering my email address:
```
events.js:187
      throw er; // Unhandled 'error' event
      ^

AssertionError [ERR_ASSERTION]: Trying to copy from a source that does not exist: /home/eric/.node_modules/lib/node_modules/generator-emacs-package/generators/common/templates/.gitignore
    at EditionInterface.exports.copy (/home/eric/.node_modules/lib/node_modules/generator-emacs-package/node_modules/mem-fs-editor/lib/actions/copy.js:48:3)
    at EditionInterface.module.exports [as copyTpl] (/home/eric/.node_modules/lib/node_modules/generator-emacs-package/node_modules/mem-fs-editor/lib/actions/copy-tpl.js:29:8)
    at module.exports.createGitignore (/home/eric/.node_modules/lib/node_modules/generator-emacs-package/generators/common/index.js:24:17)
    at Object.<anonymous> (/home/eric/.node_modules/lib/node_modules/generator-emacs-package/node_modules/yeoman-generator/lib/index.js:424:27)
    at /home/eric/.node_modules/lib/node_modules/generator-emacs-package/node_modules/run-async/index.js:25:25
    at new Promise (<anonymous>)
    at /home/eric/.node_modules/lib/node_modules/generator-emacs-package/node_modules/run-async/index.js:24:19
    at /home/eric/.node_modules/lib/node_modules/generator-emacs-package/node_modules/yeoman-generator/lib/index.js:425:13
    at processImmediate (internal/timers.js:439:21)
Emitted 'error' event on  instance at:
    at Immediate.<anonymous> (/home/eric/.node_modules/lib/node_modules/generator-emacs-package/node_modules/yeoman-generator/lib/index.js:433:22)
    at processImmediate (internal/timers.js:439:21) {
  generatedMessage: false,
  code: 'ERR_ASSERTION',
  actual: false,
  expected: true,
  operator: '=='
}
```

**Comment by EricCrosson** [comment]
Thanks for the report -- the issue is npm is ignoring the `.gitignore` file so I'll have to rename it

Funny that your name is Eric too, I saw the path in your log and grepped for hard-coded `eric`s thinking I was using an absolute path!

**Comment by EricCrosson** [comment]
I guess this isn't a problem when I install locally with `npm link` but `npm publish --dry-run` shows the .gitignore won't be pushed externally

**Comment by EricCrosson** [comment]
Should be fixed as of version 1.5.3, cheers

**Comment by berquist** [comment]
Ah, I probably could have fixed that and made a PR...thanks!
